src/workers/notifyNFTSalesWorker.ts

```diff
@@ -67,12 +62,32 @@ export default function newWorker(
           if (nftSale.buyer === project.mintAddress) {
             return;
           }
-
-          await notifyDiscordSale(discordClient, channel, nftSale);
+          if (channel) {
+            notifyDiscordSale(discordClient, channel, nftSale)
```

Comment:
I am keen to keep this synchronised for now so the twitter feature don't change the existing behaviour.